barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -40,12 +40,16 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
     std::shared_ptr<CommitmentKey> commitment_key;
     State state;
 
-    ProtoGalaxyProver_() = default;
-    ProtoGalaxyProver_(const std::vector<std::shared_ptr<Instance>>& insts)
+    ProtogalaxyProver_() = default;
+    ProtogalaxyProver_(const std::vector<std::shared_ptr<Instance>>& insts)
         : instances(ProverInstances_(insts))
         // TODO(https://github.com/AztecProtocol/barretenberg/issues/878)
         , commitment_key(instances[1]->proving_key.commitment_key){};
 
+    // Returns the accumulator, which is the first element in ProverInstances. The accumulator is assumed to have the
+    // FoldingParameters set and be the result of a previous round of folding.
+    std::shared_ptr<Instance> get_accumulator() { return instances[0]; }
+
     /**
      * @brief For each instance produced by a circuit, prior to folding, we need to complete the computation of its
```

Comment:
we kinda have the same documentation for both run_oink_prover_on_instance and run_oink_prover_on_each_instance so these could be refined. I also I'm quite meh about the function names but cant come up with sth better than `run_oink_on_all_instances` and `run_oink_on_instance`. To me I think mentioning prover is unnecessary because we are in a prover so we won't call oink_verifier but up to you,